draft-ietf-quic-transport.md

```diff
@@ -383,7 +383,7 @@ data to a peer.
 ~~~
        o
        | Create Stream (Sending)
-       | Create Bidirectional Stream (Receiving)
+       | Remote Creates Bidirectional Stream
```

Comment:
s/Remote/peer

Reply:
Change the text below as well, for transition from Ready to Send